jsonrpc/helper.go

```diff
@@ -270,3 +275,34 @@ func DecodeTxn(arg *txnArgs, store nonceGetter, forceSetNonce bool) (*types.Tran
 
 	return txn, nil
 }
+
+// expands home directory in file paths.
+// ~someuser/tmp will not be expanded.
+func expandHome(p string) string {
```

Comment:
Is this better naming for the function?
```suggestion
func resolveHomeDirectory(p string) string {
```